### ISSUE TITLE
Dicewars works pretty well

### ISSUE THREAD

**Issue by jnahmias** [ruffle-rs/ruffle#4799]
Thanks for making this project!
Just wanted to report that one of my favorites from back in the day works very well and is quite playable, though not perfectly.
https://www.gamedesign.jp/games/dicewars/flash/dice.swf

**Comment by relrelb** [comment]
Do you encounter any issues while playing this game?

**Comment by jnahmias** [comment]
Only issue is the quality of the display. Both the text and the graphical images are very fuzzy. Here's a screenshot playing it via ruffle.rs on FF 78.12.0esr:
![image](https://user-images.githubusercontent.com/1028103/127259346-26e9e6e0-1ab2-49be-8358-631134235474.png)

**Comment by arantius** [comment]
I actually came here to say a bit of the opposite:

In ruffle, the (many, very short) sounds generally don't play correctly.  When attacking, there should be many short "dits" of the dice rolling, then either a "crunch" or "ping" when the attack succeeds or fails.  The "crunch" sound is a bit longer and gets cut off when it plays (sometimes is missing completely), while the "dits" tend to sound scratchy.

Above I'm referring to a pretty old build (not sure exactly which, ~February this year).  I tried updating to the latest today and I get black rays drawn from the top left corner which don't belong (they all seem to be extensions of a short line around a hex which does belong):

![image](https://user-images.githubusercontent.com/84729/143601110-2f9d3cf9-b9ce-485f-8668-fea8303661b6.png) 

(Sounds are a little better in this latest build.  Another bug I was going to report is also fixed in this build: older ruffle only supports clicking on the dice themselves, official player lets you click anywhere in the area.)

**Comment by Lord-McSweeney** [comment]
I cannot reproduce any of the issues mentioned with the latest version of Ruffle

**Comment by arantius** [comment]
Agreed! In `ruffle-nightly-2025_09_10-web-selfhosted.zip` all issues I described above are not present.

**Comment by jnahmias** [comment]
Concur - works wonderfully! Thanks for this amazing project and the online demo at https://ruffle.rs/demo/!
